tests/drivers/build_all/audio/i2c_devices.overlay

```diff
@@ -42,6 +42,13 @@
 				status = "okay";
 				reg = <0x2>;
 			};
+
+			test_i2c_max98091: max98091@10 {
+				compatible = "maxim,max98091";
+				status = "okay";
+				reg = <0x10>;
+				mclk-frequency = <13000000>;
+			};
```

Comment:
Increment the address by 1 from the previous node. Should be `reg = <0x3>`. This is a build test that doesn't execute on real hardware, so the address doesn't need to match the datasheet.